src/suggest.c

```diff
@@ -300,7 +305,7 @@ int RSSuggestGetCommand(RedisModuleCtx *ctx, RedisModuleString **argv, int argc)
 
   Trie *tree = RedisModule_ModuleTypeGetValue(key);
   if (tree == NULL) {
-    RedisModule_ReplyWithNull(ctx);
+    RedisModule_ReplyWithSetOrArray(ctx, 0);
```

Comment:
Will `RedisModule_ReplyWithArray` suffice?

Reply:
OK, thanks. Now I'm using `RedisModule_ReplyWithArray()` for empty and non-empty results.
But  the spell check dictionary is using `RedisModule_ReplyWithSetOrArray()`, what should we use to align both commands?